Vagrantfile

```diff
@@ -129,9 +129,11 @@ module Forklift
   plugin_vagrantfiles.each { |f| load f }
   plugin_base_boxes
   unsorted = @box_loader.add_boxes('boxes.yaml', 'config/versions.yaml') if File.exists?('boxes.yaml')
-  @boxes  = unsorted.keys.sort.inject({}) do |hash, key|
-    hash[key] = unsorted[key]
-    hash
+  if unsorted
+    @boxes  = unsorted.keys.sort.inject({}) do |hash, key|
```

Comment:
I re-factored this code a few times while i was writing it, and I think i switched to inject so that i could store it all in @boxes but forgot to go back and get rid of the unsorted variable, so yes, you can use @boxes